torch/csrc/jit/python/init.cpp

```diff
@@ -1391,6 +1391,12 @@ void initJITBindings(PyObject* module) {
                 auto snb = toSymIntNode(a, b);
                 return a->mod(snb);
               })
+          .def(
+              "__rmod__",
+              [](c10::SymIntNode a, py::object b) -> c10::SymIntNode {
```

Comment:
i thought python switched the operands already.  then you'd still want a->mod(snb) wouldn't you?

Reply:
Err, no? x % y is x.__mod__(y) or y.__rmod__(y). And we are not reversing before calling into C++. And swapping is symmetric with the other r method bindings in this file

Reply:
oh, you are right.